sugoi-api-jms-store-provider/src/main/java/fr/insee/sugoi/jms/JmsWriterStore.java

```diff
@@ -18,127 +18,212 @@
+    this.queueUrgentResponseName = queueUrgentResponseName;
+  }
+
   @Override
   public void deleteUser(String id) {
-    // TODO Auto-generated method stub
-
+    Map<String, Object> params = new HashMap<>();
+    params.put("id", id);
+    params.put("realm", realm.getName());
+    params.put("userStorage", userStorage.getName());
+    jmsWriter.writeRequestInQueue(queueRequestName, "deleteUser", params);
   }
 
   @Override
   public User createUser(User user) {
-    // TODO Auto-generated method stub
-    return null;
+    Map<String, Object> params = new HashMap<>();
+    params.put("user", user);
+    params.put("realm", realm.getName());
+    params.put("userStorage", userStorage.getName());
+    jmsWriter.writeRequestInQueue(queueRequestName, "createUser", params);
+    return user;
   }
 
   @Override
   public User updateUser(User updatedUser) {
-    // TODO Auto-generated method stub
-    return null;
+    Map<String, Object> params = new HashMap<>();
+    params.put("updatedUser", updatedUser);
+    params.put("realm", realm.getName());
+    params.put("userStorage", userStorage.getName());
+    jmsWriter.writeRequestInQueue(queueRequestName, "updateUser", params);
+    return updatedUser;
   }
 
   @Override
   public void deleteGroup(String appName, String groupName) {
-    // TODO Auto-generated method stub
-
+    Map<String, Object> params = new HashMap<>();
+    params.put("appName", appName);
+    params.put("groupName", groupName);
+    params.put("realm", realm.getName());
+    params.put("userStorage", userStorage.getName());
+    jmsWriter.writeRequestInQueue(queueRequestName, "deleteGroup", params);
   }
 
   @Override
   public Group createGroup(String appName, Group group) {
-    // TODO Auto-generated method stub
-    return null;
+    Map<String, Object> params = new HashMap<>();
+    params.put("appName", appName);
+    params.put("group", group);
+    params.put("realm", realm.getName());
+    params.put("userStorage", userStorage.getName());
+    jmsWriter.writeRequestInQueue(queueRequestName, "createGroup", params);
+    return group;
   }
 
   @Override
   public Group updateGroup(String appName, Group updatedGroup) {
-    // TODO Auto-generated method stub
-    return null;
+    Map<String, Object> params = new HashMap<>();
+    params.put("appName", appName);
+    params.put("updatedGroup", updatedGroup);
+    params.put("realm", realm.getName());
+    params.put("userStorage", userStorage.getName());
+    jmsWriter.writeRequestInQueue(queueRequestName, "updateGroup", params);
+    return updatedGroup;
   }
 
   @Override
   public void deleteOrganization(String name) {
-    // TODO Auto-generated method stub
-
+    Map<String, Object> params = new HashMap<>();
+    params.put("name", name);
+    params.put("realm", realm.getName());
+    params.put("userStorage", userStorage.getName());
+    jmsWriter.writeRequestInQueue(queueRequestName, "deleteOrganization", params);
```

Comment:
All the keys used here (and in the receiver) should be enums I think (or constants) to avoid trivial typo.

Reply:
done !!